.travis.yml

```diff
@@ -30,8 +30,8 @@ matrix:
   include:
```

Comment:
GA also test against `PROFILE=''`

Reply:
so,  add tags into profile instead of EXECLUDE_TAGS?

Reply:
PROFILE='' == PROFILE="-Pspark-3.0", this pr seems to delete the default job?

Reply:
ok, got it